EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76943
+graphicux.com#$#body { overflow: scroll !important; }
```

Comment:
It doesn't look like an anti-adblock. It seems that it's incorrectly blocked annoyance.
Please remove this rule and change current one:
https://github.com/AdguardTeam/AdguardFilters/blob/77a12509147dee49f5511afc7495288a8bc2817a/AnnoyancesFilter/sections/popups.txt#L257
to for example, something like this:
```
lalanguefrancaise.com,graphicux.com#$#html[class].hustle-no-scroll { overflow: auto !important; }
```